ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -154,44 +165,72 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
     return;
   }
 
+  setTierRequirement(membershipRequirement, attribute) {
+    // TODO(RW-6933): Implement new institution admin UI with CT support.
+    this.setState(fp.set(['institution', attribute],
+      [{accessTierShortName: AccessTierShortNames.Registered,
+        membershipRequirement: membershipRequirement.value, eRARequired: true}]));
+
+    // Dual write DuaTypeEnum for now. Because account creation flow still uses that field.
+    const duaType = membershipRequirement.value === InstitutionMembershipRequirement.ADDRESSES
+        ? DuaType.RESTRICTED : DuaType.MASTER;
+    this.setState(fp.set(['institution', 'duaTypeEnum'], duaType));
+  }
+
   setEmails(emailInput, attribute) {
```

Comment:
I think `attribute` is no longer necessary, can just be a hrdcoded property in this function